File: datasets/IR-Plag/original-java-rest-kotlin/case-06/plagiarized/L4/08/Kasus6L4.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus6L4 {
    @JvmStatic
    fun main(args: Array<String>) {
        //Delarasi
        val arrNum = IntArray(10)
        val inp = Scanner(System.`in`)
        // for untuk 10 kali input
        for (i in 0..9) {
            // Read a number
            print("Read a number: ")
            arrNum[i] = inp.nextInt()
        }
        // Display the array
        Reverse(arrNum)
    }

    private fun Reverse(arr: IntArray) {
        for (i in 9 downTo 0) {
            //Print angka dalam array
            println(arr[i])
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L4/08/Kasus6L4.ts
import { Scanner } from 'readline-sync';

function Reverse(arr: number[]): void {
  for (let i = 9; i >= 0; i--) {
    console.log(arr[i]);
  }
}

function main(): void {
  const arrNum: number[] = new Array(10);
  const inp: Scanner = new Scanner();

  for (let i = 0; i < 10; i++) {
    console.log('Read a number: ');
    arrNum[i] = inp.nextInt();
  }

  Reverse(arrNum);
}

main();

